src/overlays/actors/ovl_Dm_Stk/z_dm_stk.c

```diff
@@ -1967,13 +1967,13 @@ void DmStk_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot,
                     (this->objectStk2ObjectSlot > OBJECT_SLOT_NONE)) {
                     Matrix_Push();
                     Matrix_Scale(2.0f, 2.0f, 2.0f, MTXMODE_APPLY);
-                    gSegments[6] = OS_K0_TO_PHYSICAL(play->objectCtx.slots[this->objectStk2ObjectSlot].segment);
+                    gSegments[0x06] = OS_K0_TO_PHYSICAL(play->objectCtx.slots[this->objectStk2ObjectSlot].segment);
 
                     gSPSegment(POLY_OPA_DISP++, 0x06, play->objectCtx.slots[this->objectStk2ObjectSlot].segment);
 
                     AnimatedMat_Draw(play, Lib_SegmentedToVirtual(gSkullKidMajorasMaskCurseOverlayTexAnim));
                     Gfx_DrawDListOpa(play, gSkullKidMajorasMaskCurseOverlayDL);
-                    gSegments[6] = OS_K0_TO_PHYSICAL(play->objectCtx.slots[this->objectStkObjectSlot].segment);
+                    gSegments[0x06] = OS_K0_TO_PHYSICAL(play->objectCtx.slots[this->objectStkObjectSlot].segment);
 
                     gSPSegment(POLY_OPA_DISP++, 0x06, play->objectCtx.slots[this->objectStkObjectSlot].segment);
```

Comment:
is it the standard in MM to use `0x0N` for segment number?

since they're 0-0xF I would not have put the extra `0`
I'm not even sure I would have done 0xN for 0-9

is this choice motivated or just how it is?

Reply:
I kind of remember a small comment PR thread about this where I think someone (@Thar0 perhaps, terribly sorry if I am misremembering) preferred this style. Not quite sure on the specifics though.

Reply:
Well if someone knows I'd like to know

Don't let this block merging btw I'm just curious, OoT is inconsistent on this and I don't have a strong opinion on what to standardize on

Reply:
I've seen it as one of those unspoken consistency things where that's the style I've seen most common, but I also don't have a strong opinion if it were to change